.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
```

Comment:
Just that the crate was created before cargo started adding one on crate creation :-)